pkg/execution/runner/runner.go

```diff
@@ -636,15 +640,27 @@ func (s *svc) initialize(ctx context.Context, fn inngest.Function, evt event.Tra
 			return err
 		}
 		if limited {
+			if evt.GetEvent().Name == event.InvokeFnName {
```

Comment:
I know it's small, but can we make this a method on `event.Event`? Like `IsInvocationEvent()`.

We've discussed making some internal events sendable before and, while we're not doing that now, it'll make these checks a lot easier.